assignments/subscriptions/1baf5470-1c3e-40d3-a6f7-74bfbce4b348/assign.tagging.json

```diff
@@ -0,0 +1,31 @@
+{
+    "properties": {
+      "metadata": {
+        "assignedBy": "PlatOps"
+      },
+      "description": "Policy to check required tags",
+      "displayName": "HMCTS Tagging - CFT PTL",
+      "enforcementMode": "DoNotEnforce",
+      "nonComplianceMessages": [
+        {
+            "message": "Resource doesn't meet the tagging requirements. See https://github.com/hmcts/azure-policy/blob/master/policies/tagging/README.md"
+        }
+      ],
+      "notScopes": [
+        "/subscriptions/1baf5470-1c3e-40d3-a6f7-74bfbce4b348/resourceGroups/cft-ptl-00-aks-node-rg",
+        "/subscriptions/1baf5470-1c3e-40d3-a6f7-74bfbce4b348/resourceGroups/cft-ptl-01-aks-node-rg",
+        "/subscriptions/1baf5470-1c3e-40d3-a6f7-74bfbce4b348/resourceGroups/cft-ptl-00-rg",
+        "/subscriptions/1baf5470-1c3e-40d3-a6f7-74bfbce4b348/resourceGroups/cft-ptl-01-rg"
+      ],
+      "parameters": {},
+      "policyDefinitionId": "/providers/Microsoft.Management/managementGroups/dts002/providers/Microsoft.Authorization/policyDefinitions/HMCTSTagging",
+      "scope": "/subscriptions/1baf5470-1c3e-40d3-a6f7-74bfbce4b348"
+    },
+    "sku": {
+      "name": "A0",
+      "tier": "Free"
+    },
+    "type": "Microsoft.Authorization/policyAssignments",
+    "id": "/subscriptions/1baf5470-1c3e-40d3-a6f7-74bfbce4b348/providers/Microsoft.Authorization/policyAssignments/HMCTSTagging_DTS-CFTPTL-INTSVC",
+    "name": "HMCTSTagging_DTS-CFTPTL-INTSVC"
```

Comment:
Looking at this and the other assignments below and thinking, should the name include the subscription name or just the environment name. In this case the subscription DTS-CFTPTL-INTSVC is referred to as PTL and is commonly known as that so makes sense to use this to me.

be interesting to see if others agree here or not.

Reply:
I thought about that but kept following the other assignments that had the actual subscription name appended to the name and ID. I did try to put the names we generally refer to in the "Display Name" field. But I'm happy to change if most think it should be reflected on the name too.

Reply:
I think this is fine as displayName `HMCTS Tagging - CFT PTL` shows a friendly reader name